### PR TITLE
feat(READ.ME): Update README.md project document

### PR DESCRIPTION
**Pull request by gsi-juandavid**

### CODE REVIEW

**Review by gsi-dayanacaridad — APPROVED** [review]

**Review by gsi-dayanacaridad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,6 +2,64 @@
...
+You can create a new set of ClientId and ClientSecret, by registering and logging in https://test.deribit.com/. Visit the API documentation in https://docs.deribit.com/. 
+
+    "ClientId": "[Your client ID here]",
+    "ClientSecret": "[Your client secret here]", 
+
+    "WebSocketUrl": "wss://test.deribit.com/ws/api/v2",
+The default value of this setting is wss://test.deribit.com/ws/api/v2, which is the test environment of the Deribit API. If you want to connect to a different environment, you must set this value accordingly.
+
+
+    "ConnectionTimeoutInMilliseconds": 5000,
```
Comment (gsi-yoancme) [inline README.md:24]: this will be no needed anymore

File: README.md [hunk 2]
```diff
@@ -62,4 +120,14 @@ Besides, the team gave each of the effort tags a corresponding amount of hours.
 &nbsp;
 
 
-## How can I collaborate (Git checklist)
+## How can I collaborate
+
+You can expand the code base of this project. To contribute code to the project, you should: 
+
+1- Create a feature branch
```
Comment (gsi-yoancme) [inline README.md:127]: or a repository fork

**Review by gsi-yoancme — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (gsi-juandavid) [reply README.md:24]: ok

File: README.md (hunk 2 above)
Comment (gsi-juandavid) [reply README.md:127]: ok

**Review by gsi-robertogarcia — APPROVED** [review]

**Review by gsi-dayanacaridad — APPROVED** [review]

**Review by gsi-juandavid — COMMENTED** [review]
DONE

**Pull request merged**